content/docs/integrations/vercel.md

```diff
@@ -1,43 +1,63 @@
 ---
-title: Run a Vercel and Next.js app
+title: Run a Next.js app
 enableTableOfContents: true
 redirectFrom:
   - /docs/quickstart/vercel
 ---
 
-## Introduction
+Next.js by Vercel is an open-source web development framework that enables React-based web applications. This topic describes how to create a Neon project and access it from a Next.js app.
 
-Vercel is a serverless platform used by developers for web hosting and other web services with the benefit of automatic scaling and smooth deployment; it also allows you to deploy your Next.js applications with full-on support.
+To create a Neon project and access it from a Next.js app:
 
-## Create a Next.js project
-
-First, [Create a next.js project](https://nextjs.org/learn/basics/create-nextjs-app/setup) if you don't have one.
+1. [Create a Neon project](#create-a-neon-project)
+2. [Create a Next.js project](#create-a-nextjs-project)
+3. [Add a PostgreSQL client to your app](#add-a-postgresql-client-to-your-app)
+4. [Add your Neon connection details](#add-your-neon-connection-details)
+5. [Connect to the Neon database](#connect-to-the-neon-database)
 
 ## Create a Neon project
 
-Next, create a Neon Project for your app. You can configure your database schema from Neon Console or using tools like Prisma.
+When creating a Neon project, take note of your project ID, database name, user, password, and port number. This information is required when defining your connection settings. 
+
+To create a Neon project:
 
-## Add PostgreSQL client
+1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
+2. Click **New Project**.
+3. Enter a name for your project and click **Create Project**.
 
-Add PostgreSQL client to your app. In this example we used [postgres.js](https://www.npmjs.com/package/postgres), but feel free to choose another one.
+For additional information about creating projects, see [Setting up a project](/docs/get-started-with-neon/setting-up-a-project).
+
+## Create a Next.js project
 
-## Add Neon credentials
+Create a Next.js project if you do not have one. For instructions, see [Create a Next.js App](https://nextjs.org/learn/basics/create-nextjs-app/setup), in the Vercel documentation.
 
-Add your Neon credentials to the `.env` file.
+## Add a PostgreSQL client to your app
+
+Add a PostgreSQL client to your app, such as `Postgres.js`.  For instructions, refer to the [postgres.js Getting started](https://www.npmjs.com/package/postgres).
+
+## Add your Neon connection details
+
+Add your Neon connection details to your `.env` file.
 
 ```shell
-NEON_HOST=...
-NEON_DB=...
-NEON_USER=...
-NEON_PASS=...
-NEON_PORT=...
+NEON_HOST='<project_id>.cloud.neon.tech'
```

Comment:
I think connection string method is rather easier to use

Reply:
Hi @duskpoet 
Happy to use the connection string instead. In this case, do I simply drop the connection string with the password included in the .env file as provided on the Dashboard?

And in that case, how should I modify the connection snippet that we provide?

import postgres from 'postgres';

const sql = postgres({
  host: process.env.NEON_HOST,
  port: process.env.NEON_PORT,
  database: process.env.NEON_DB,
  username: process.env.NEON_USER,
  password: process.env.NEON_PASS,
});

const result = await sql.uafe(req.body);

Reply:
.env
```
DATABASE_URL=<neon_connection_string>
```
index.ts
```
const sql = postgres(process.env.DATABASE_URL);
```

Reply:
Thank you!